tests/topologyDescription/topologyDescription-hasReadableServer_error-001.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+MongoDB\Driver\TopologyDescription::hasReadableServer() (argument with bad type)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
```

Comment:
This test will need to be split for PHP 7 and 8 since the error messages differ. See `session-startTransaction_error-004.phpt` and `session-startTransaction_error-005.phpt` for prior art. You can lift the same skip logic for PHP versions from those tests.

Reply:
Alternatively, just remove the `echo` and trailing newline from `throws()` output and we can make this portable for both PHP versions.